compiler/rustc_codegen_llvm/src/llvm_util.rs

```diff
@@ -218,13 +218,34 @@ pub fn target_cpu(sess: &Session) -> &str {
     handle_native(name)
 }
 
-pub fn handle_native_features(sess: &Session) -> Vec<String> {
+/// The list of LLVM features computed from CLI flags (`-Ctarget-cpu`, `-Ctarget-feature`,
+/// `--target` and similar).
+// FIXME(nagisa): Cache the output of this somehow? Maybe make this a query? We're calling this
+// for every function that has `#[target_feature]` on it. The global features won't change between
```

Comment:
was this function called as often before, by the way ? we should probably run the perfbot on this PR, as there are thousands of vendor and SIMD intrinsics with `#[target_feature]` in `stdarch`, just in case this has become more expensive.